prog-common.go

```diff
@@ -809,6 +811,20 @@ const (
 	BPFAttachTypeSKReusePortSelectorMigrate BPFAttachType = C.BPF_SK_REUSEPORT_SELECT_OR_MIGRATE
 	BPFAttachTypePerfEvent                  BPFAttachType = C.BPF_PERF_EVENT
 	BPFAttachTypeTraceKprobeMulti           BPFAttachType = C.BPF_TRACE_KPROBE_MULTI
+	BPFAttachTypeLSMCgroup                  BPFAttachType = C.BPF_LSM_CGROUP
+	BPFAttachTypeStructOps                  BPFAttachType = C.BPF_STRUCT_OPS
+	BPFAttachTypeNetfilter                  BPFAttachType = C.BPF_NETFILTER
+	BPFAttachTypeTCXIngress                 BPFAttachType = C.BPF_TCX_INGRESS
+	BPFAttachTypeTCXEgress                  BPFAttachType = C.BPF_TCX_EGRESS
+	BPFAttachTypeTraceUprobeMulti           BPFAttachType = C.BPF_TRACE_UPROBE_MULTI
+	BPFAttachTypeCgroupUnixConnect          BPFAttachType = C.BPF_CGROUP_UNIX_CONNECT
+	BPFAttachTypeCgroupUnixSendMsg          BPFAttachType = C.BPF_CGROUP_UNIX_SENDMSG
+	BPFAttachTypeCgroupUnixRecvMsg          BPFAttachType = C.BPF_CGROUP_UNIX_RECVMSG
+	BPFAttachTypeCgroupUnixGetPeerName      BPFAttachType = C.BPF_CGROUP_UNIX_GETPEERNAME
+	BPFAttachTypeCgroupUnixGetSockName      BPFAttachType = C.BPF_CGROUP_UNIX_GETSOCKNAME
+	BPFAttachTypeNetkitPrimary              BPFAttachType = C.BPF_NETKIT_PRIMARY
+	BPFAttachTypeNetkitPeer                 BPFAttachType = C.BPF_NETKIT_PEER
+	BPFAttachTypeTraceKprobeSession         BPFAttachType = C.BPF_TRACE_KPROBE_SESSION
```

Comment:
Its seems is missing the last one:
```
BPF_TRACE_UPROBE_SESSION
```
https://elixir.bootlin.com/linux/v6.13/source/include/uapi/linux/bpf.h#L1119